docs/_user_guide/slurm.md

```diff
@@ -133,50 +129,49 @@ The four partitions have the following limits:
 
 ### How do I check how busy the machines are?
 
 You can pass format options to the `sinfo` command as follows:
 
-```bash
+```{.yaml .no-copy title="Terminal Output"}
 USER@yen4:~$ sinfo --format="%m | %C"
 MEMORY | CPUS(A/I/O/T)
-257366+ | 100/1148/0/1248
+257366+ | 855/713/0/1568
 ```
 
-where `MEMORY` outputs the minimum size of memory of the `yen-slurm` cluster node in megabytes (256 GB) and
-`CPUS(A/I/O/T)` prints the number of CPU's that are allocated / idle / other / total.
-For example, if you see `100/1148/0/1248` that means 100 CPU's are allocated, 1,148 are idle (free) out of 1,248 CPU's total.
+where MEMORY outputs the minimum size of memory of the `yen-slurm` cluster node in megabytes (256 GB) and CPUS(A/I/O/T) prints the number of CPU's that are allocated / idle / other / total.
+For example, if you see `855/713/0/1568` that means 855 CPU's are allocated, 713 are idle (free) out of 1,568 CPU's total.
 
 You can also run `checkyens` and look at the last line for summary of all pending and running jobs on yen-slurm.
 
-```bash
+```{.yaml .no-copy title="Terminal Output"}
 USER@yen4:~$ checkyens
 Enter checkyens to get the current server resource loads. Updated every minute.
-yen1 :  5 Users | CPU [                     3%] | Memory [#                    8%] | updated 2023-01-13-09:44:01
-yen2 :  2 Users | CPU [                     0%] | Memory [                     0%] | updated 2023-01-13-09:44:02
-yen3 :  2 Users | CPU [                     0%] | Memory [##                  14%] | updated 2023-01-13-09:44:01
-yen4 :  3 Users | CPU [#####               29%] | Memory [###                 15%] | updated 2023-01-13-09:44:01
-yen5 :  1 Users | CPU [##                  10%] | Memory [######              31%] | updated 2023-01-13-09:44:01
-yen-slurm : 12 jobs, 0 pending | 86 CPUs allocated (21%) | 300G Memory Allocated (5%) | updated 2023-01-13-09:44:02
+yen1 :  2 Users | CPU [#######             36%] | Memory [####                22%] | updated 2024-10-22-00:16:01 
+yen2 :  3 Users | CPU [                     1%] | Memory [########            42%] | updated 2024-10-22-00:16:00 
+yen3 :  6 Users | CPU [                     1%] | Memory [###                 15%] | updated 2024-10-22-00:16:00 
+yen4 :  2 Users | CPU [                     0%] | Memory [###                 16%] | updated 2024-10-22-00:16:00 
+yen5 :  0 Users | CPU [####                21%] | Memory [####                21%] | updated 2024-10-22-00:16:04 
+yen-slurm : 386 jobs, 7 pending | 834 CPUs allocated (53%) | 5142G Memory Allocated (50%) | updated 2024-10-22-00:16:02
 ```
 
 ### When will my job start?
 
 You can ask the scheduler using `squeue --start`, and look at the `START_TIME` column.
 
-```bash
+```{.yaml .no-copy title="Terminal Output"}
 USER@yen4:~$ squeue --start
 
```

Comment:
I would add a note after the printout to say that start times are conservative and are based on max duration of every job in the queue so it is likely that the job just submitted will start sooner than this predicts